static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
Thanks @guybedford , I haven't tried your suggestion yet, but I'll give it a go tomorrow and let you know.